RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitProducerGPU.cc

```diff
@@ -205,45 +215,45 @@ void EcalUncalibRecHitProducerGPU::acquire(edm::Event const& event,
   }
 
   // conditions
-  setup.get<EcalPedestalsRcd>().get(pedestalsHandle_);
-  setup.get<EcalGainRatiosRcd>().get(gainRatiosHandle_);
-  setup.get<EcalPulseShapesRcd>().get(pulseShapesHandle_);
-  setup.get<EcalPulseCovariancesRcd>().get(pulseCovariancesHandle_);
-  setup.get<EcalSamplesCorrelationRcd>().get(samplesCorrelationHandle_);
-  setup.get<EcalTimeBiasCorrectionsRcd>().get(timeBiasCorrectionsHandle_);
-  setup.get<EcalTimeCalibConstantsRcd>().get(timeCalibConstantsHandle_);
-  setup.get<EcalSampleMaskRcd>().get(sampleMaskHandle_);
-  setup.get<EcalTimeOffsetConstantRcd>().get(timeOffsetConstantHandle_);
-  setup.get<EcalMultifitParametersGPURecord>().get(multifitParametersHandle_);
-
-  auto const& pedProduct = pedestalsHandle_->getProduct(ctx.stream());
-  auto const& gainsProduct = gainRatiosHandle_->getProduct(ctx.stream());
-  auto const& pulseShapesProduct = pulseShapesHandle_->getProduct(ctx.stream());
-  auto const& pulseCovariancesProduct = pulseCovariancesHandle_->getProduct(ctx.stream());
-  auto const& samplesCorrelationProduct = samplesCorrelationHandle_->getProduct(ctx.stream());
-  auto const& timeBiasCorrectionsProduct = timeBiasCorrectionsHandle_->getProduct(ctx.stream());
-  auto const& timeCalibConstantsProduct = timeCalibConstantsHandle_->getProduct(ctx.stream());
-  auto const& multifitParametersProduct = multifitParametersHandle_->getProduct(ctx.stream());
+  auto const& pedestalsHandle = setup.getHandle(pedestalsToken_);
```

Comment:
You can skip the handle with
```suggestion
  auto const& pedProduct = setup.getData(pedestalsToken_)->getProduct(ctx.stream());
```